dask/dataframe/core.py

```diff
@@ -4132,6 +4191,45 @@ def applymap(self, func, meta="__no_default__"):
     def round(self, decimals=0):
         return elemwise(M.round, self, decimals)
 
+    @derived_from(pd.DataFrame)
+    def mode(self, axis=0, numeric_only=False, dropna=True, split_every=False):
```

Comment:
Hmm why are you defining this on all of _Frame, Series, and DataFrame? I don't think you'd need all three.

Reply:
Now mode is nontrivially defined for _Frame and DataFrame.  The Series mode merely calls _Frame mode.